core/src/main/resources/application.conf

```diff
@@ -118,6 +120,11 @@ call-caching {
   // request to an external service (DockerHub, GCR).  If a call fails to lookup a
   // Docker hash, it will fail.
   lookup-docker-hash = false
+
+  // This option determines whether file contents are hashed to determine cache hit or miss.
+  hash-file-contents = true
+  // This option determines whether file paths are hashed to determine cache hit or miss.
+  hash-file-paths = false
```

Comment:
Actually, 'off' doesn't really make sense for either of these, does it?  So perhaps default to "name" and "path" respectively, with the option to upgrade either to "lookup".